app/models/concerns/project_search.rb

```diff
@@ -1,5 +1,11 @@
 # frozen_string_literal: true
 module ProjectSearch
+  # Whenever a PackageManager::Base platform is removed, it's easier for now to
+  # just add it to this list, until the records/indices are cleared out of that platform.
+  # NB these are the casings from the database, which sometimes don't match the PackageManager::Base
+  # formatted_name casings, e.g. Pypi vs PyPI
```

Comment:
Ideally we could just safelist all platforms from `PackageManager::Base.platforms.map(&:formatted_name)` but they don't always map 1:1 with `projects.platform` in the database, so we can keep this explicit blocklist array here instead.